src/client/components/forms/userCollection.js

```diff
@@ -0,0 +1,306 @@
+/*
+ * Copyright (C) 2020 Prabal Singh
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+
+import * as bootstrap from 'react-bootstrap';
+import CustomInput from '../../input';
+import EntitySearchFieldOption from '../../entity-editor/common/entity-search-field-option';
+import PropTypes from 'prop-types';
+import React from 'react';
+import ReactSelect from 'react-select';
+import SelectWrapper from '../input/select-wrapper';
+import classNames from 'classnames';
+import request from 'superagent-bluebird-promise';
```

Comment:
I just merged #438 which updates a few packages, including superagent that now doesn't need the extra 'superagent-bluebird-promise' package anymore.

You will also need to remove the `.promise()` call after `request.post…`.
Here's an example: https://github.com/bookbrainz/bookbrainz-site/pull/438/files#diff-e8473140e224bec0384c75907f7aee19